### PR TITLE
OCP : cleanup, bump

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: media-sound/ocp/ocp-0.1.22_git.recipe [hunk 1]
```diff
@@ -1,70 +1,83 @@
...
+	libtoolize -vfi
+	chmod +x configure
+	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
+	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
+	chmod +x configure $commandBinDir/{ocp-curses,ocp-sdl,ultrafix.sh}
```
Comment (fbrosson) [inline media-sound/ocp/ocp-0.1.22_git.recipe]:
```diff
-	chmod +x configure $commandBinDir/{ocp-curses,ocp-sdl,ultrafix.sh}
+	chmod +x $commandBinDir/{ocp-curses,ocp-sdl,ultrafix.sh}
```
(I guess you forgot to remove the `configure` from your copy/paste.)

File: media-sound/ocp/ocp-0.1.22_git.recipe [hunk 2]
```diff
@@ -1,70 +1,83 @@
...
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
 PROVIDES="
 	ocp$secondaryArchSuffix = $portVersion
-	cmd:ocp = $portVersion
-	cmd:ocp_0.1.21 = $portVersion
+	cmd:ocp$commandSuffix = $portVersion
+	cmd:ocp_0.1.22$commandSuffix = $portVersion
```
Comment (fbrosson) [inline media-sound/ocp/ocp-0.1.22_git.recipe]: If you can rename the recipe to `ocp-0.1.22~git.recipe` and:
```diff
-	cmd:ocp_0.1.22$commandSuffix = $portVersion
+	cmd:ocp_${portVersion#~*}$commandSuffix
```
(I did not test this.)

File: media-sound/ocp/ocp-0.1.22~git.recipe [hunk 3]
```diff
@@ -1,70 +1,83 @@
...
-SOURCE_URI="git://git.code.sf.net/p/opencubicplayer/code#f97d87f960624de013826e9c2970404f90eef211"
-REVISION="1"
+COPYRIGHT="1994-2016 Niklas Beisert, Stian Skjelstad and others"
 LICENSE="GNU GPL v2"
-COPYRIGHT="1994-2012 Niklas Beisert, Stian Skjelstad and others"
-##		media-libs/flac >= 1.2.1"
-## ncurses zlib ... cf. ocp.spec
+REVISION="1"
+gitRevision="835bf803f590512848282bbf74986ebfaec17da3"
+SOURCE_URI="https://sourceforge.net/code-snapshots/git/o/op/opencubicplayer/code.git/opencubicplayer-code-$gitRevision.zip"
```
Comment (alaviss) [inline media-sound/ocp/ocp-0.1.22~git.recipe]: Official release tarballs are available at https://sourceforge.net/projects/opencubicplayer/files/ocp-0.1.21/

Comment (extrowerk) [reply media-sound/ocp/ocp-0.1.22~git.recipe]: Thats from 2011, the git source have changes from 2016. If one want, can change it to the release, i don't see the point right now.

**Comment by korli** [comment]
Please consider fixing http://vmpkg.haiku-os.org/master/x86_64/buildruns/966/builds/9993.log

**Comment by extrowerk** [comment]
@korli: for me it seems the checksum is correct.

**Pull request merged**